Doc/library/typing.rst

```diff
@@ -39,12 +39,19 @@ provides backports of these new features to older versions of Python.
 For a summary of deprecated features and a deprecation timeline, please see
 `Deprecation Timeline of Major Features`_.
 
+.. seealso::
+
+   The documentation at https://mypy.readthedocs.io/ provides accessible
+   descriptions of various type system features. Particularly refer to the "Type
```

Comment:
```suggestion
   descriptions of various type system features, although parts of the documentation
   may apply only to mypy and not to other type chekers. Particularly refer to the "Type
```
Not sure of the exact wording, but I want to make sure we don't imply that mypy is necessarily the correct type checker: it's one of many.

Reply:
I agree that's an important concern.

I'm not sure I understand what the exact thing you're suggesting is that isn't covered by the second sentence. Or is it just a matter of reordering to change emphasis?

Reply:
The existing second sentence is too implicit for my taste; it implies that some of mypy's behavior is nonstandard but doesn't outright say it.

Reply:
Makes sense, does this address:
> since the Python typing system is
   standardised via PEPs, this information should broadly apply to most Python
   type checkers, although some details may be mypy specific.

Reply:
I think I prefer Jelle's suggestion — it's nice to have the "warning" in the first sentence in my opinion.

Having the explicit warning in the second sentence could also be misread to imply that it's only the "Type system reference" section of the docs that have mypy-specific bits in it, whereas obviously it's the other bits that are actually more likely to be mypy-specific

Reply:
Hm, okay so that falls under the "reordering to change emphasis" category. I'd like to keep the link upfront without making it too scary or caveating things that we will caveat again in the next sentence.

New proposal is just get rid of first sentence:

>   The "Type System Reference" section of https://mypy.readthedocs.io/ -- since
   the Python typing system is standardised via PEPs, this information should
   broadly apply to most Python type checkers, although parts may still be mypy
   specific.

Reply:
Sounds good